boards/siyi/n7/nuttx-config/include/board_dma_map.h

```diff
@@ -0,0 +1,42 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2020 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2023 PX4 Development Team. All rights reserved.
```

If you don't mind I would do a quick pass to set these to the current year, personally I use it as a reference for when stuff was added.

Reply:
> 

Thank you very much for your review. If possible, could you help us correct any incorrect parts? Thank you. Additionally, please let us know which content we need to fix ourselves, and we will make the corrections immediately.